### DISCUSSION TITLE
Discovery kit script failure

### DISCUSSION THREAD

**Discussion by josh25219** [polarfire-soc/polarfire-soc-documentation#497] [Reference designs, FPGA and development kits]
Hi,
If i download the discovery kit release 2024.06 and run the script to build the design using Libero 2023.1, the script fails with lots of errors.

Error: Failure when executing Tcl script. [ Line 103: ERROR: Invalid parameter entry "DDR3_ADVANCED_CA_TRAINING                           false" encountered while parsing configuration file.
         ERROR: Invalid parameter entry "DDR3_DEGREE0                                        0" encountered while parsing configuration file.
         ERROR: Invalid parameter entry "DDR3_DEGREE45                                       1" encountered while parsing configuration file.
         ERROR: Invalid parameter entry "DDR3_DEGREE90                                       2" encountered while parsing configuration file.

and many more errors just like above.

Am i doing something wrong?   I just execute the script with no additional parameters.    

Thanks,
Josh

**Comment by hughbreslin** [comment]
Hi @josh25219 I don't think you're doing anything wrong. As far as I remember they're newer parameters, could you test with a newer version of Libero (2024.1 for example) and see. Potentially the minimum version of the scripts needs to be updated if those parameters aren't backwards compatible.

**Comment by josh25219** [comment]
Hi Hugh,

ahhhh yes,  it runs with Libero 2024.1.   I generally run the script with whatever version of Libero is specified (in this case 2023.1) to avoid issues with the IP core versions not matching up.    So it runs with 2024.1, but not 2023.1.   

Thanks,
Josh
